webrender/src/render_backend.rs

```diff
@@ -416,6 +427,10 @@ impl RenderBackend {
         self.frame.discard_frame_state_for_pipeline(pipeline_id);
     }
 
+    fn device_pixel_ratio(&self) -> f32 {
```

Comment:
Sorry, for the disappearing comment. I'm not sure what happened, but I've tried to reproduce it above.